File: stsWorkspace/node01/ex10.js
function Func01() {
  this.su1 = 1111;
}
console.log(new Func01().su1);
class Func02 {
  su1 = 1111;
  func1 = function () {
    console.log("메서드", this);
  };
  func3 = () => console.log("메서드", this);
  func2(a) {
    console.log("메서드", a, this);
  }
  constructor(a) {
    this.su2 = a;
  }
}
// console.log(new Func02(3333));
new Func02(3333).func2(1111);

File: javaWorkspace/Day05/src/com/am/Ex10.java
package com.am;

public class Ex10 {
//	public static int su1=1111;
	public static int su2=2222;
	public int su3 = 3333;
	
	public static void main(String[] args) {
		int su1=1111;
		System.out.println("main su1="+su1);
		func01(su1);
//		func01();
		System.out.println("main su1="+su1);
		System.out.println("main su2="+su2);
		Ex10 me = new Ex10();
		System.out.println("main su3="+me.su3);
		me.func02();
		Ex10 me2 = new Ex10();
		System.out.println("main su3="+me2.su3);
		System.out.println("main su3="+me.su3);
	}
	
	public void func02() {
		su3++;
		System.out.println("f2 su3="+su3);
	}

	public static void func01(int su1) {
		su1++;
		su2++;
		
		System.out.println("func01 su="+su1);
	}
}
